server/src/main/scala/controllers/application/api/v1/FlightsApiController.scala

```diff
@@ -63,7 +63,7 @@ class FlightsApiController @Inject()(cc: ControllerComponents, ctrl: DrtSystemIn
     }
 
   def populateFlights(start: String, end: String): Action[AnyContent] =
-    authByRole(ApiQueueAccess) {
+    authByRole(SuperAdmin) {
```

Comment:
is this temporary or you meant to keep SuperAdmin ?

Reply:
This is an admin endpoint to populate data so it needs to be SuperAdmin